.gitpod.Dockerfile

```
@@ -1,30 +1,26 @@
-FROM gitpod/workspace-full:2023-01-16-03-31-28
-
-RUN mkdir -p ~/.local/bin
-RUN curl -L https://github.com/stellar/soroban-tools/releases/download/v20.3.0/soroban-cli-20.3.0-x86_64-unknown-linux-gnu.tar.gz | tar xz -C ~/.local/bin soroban
-RUN chmod +x ~/.local/bin/soroban
-RUN curl -L https://github.com/mozilla/sccache/releases/download/v0.3.3/sccache-v0.3.3-x86_64-unknown-linux-musl.tar.gz | tar xz --strip-components 1 -C ~/.local/bin sccache-v0.3.3-x86_64-unknown-linux-musl/sccache
-RUN chmod +x ~/.local/bin/sccache
-
-RUN curl -L https://github.com/watchexec/cargo-watch/releases/download/v8.1.2/cargo-watch-v8.1.2-x86_64-unknown-linux-gnu.tar.xz | tar xJ --strip-components 1 -C ~/.local/bin cargo-watch-v8.1.2-x86_64-unknown-linux-gnu/cargo-watch
-
-ENV RUSTC_WRAPPER=sccache
-ENV SCCACHE_CACHE_SIZE=5G
-ENV SCCACHE_DIR=/workspace/.sccache
+FROM gitpod/workspace-full:latest
 
 # Remove the existing rustup installation before updating due to:
 # https://github.com/gitpod-io/workspace-images/issues/933#issuecomment-1272616892
 RUN rustup self uninstall -y
 RUN rm -rf .rustup
 RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- --default-toolchain none -y
 
-RUN rustup install 1.69
-RUN rustup target add --toolchain 1.69 wasm32-unknown-unknown
-RUN rustup component add --toolchain 1.69 rust-src
-RUN rustup default 1.69
+RUN rustup install stable
+RUN rustup target add --toolchain stable wasm32-unknown-unknown
+RUN rustup component add --toolchain stable rust-src
+RUN rustup default stable
```

We need to stay below 1.82 until this issue is resolved:
- https://github.com/stellar/rs-soroban-env/issues/1469
```suggestion
RUN rustup install 1.81
RUN rustup target add --toolchain 1.81 wasm32-unknown-unknown
RUN rustup component add --toolchain 1.81 rust-src
RUN rustup default 1.81
```